### PR TITLE
Add .nomedia file to Android assets

### PR DESCRIPTION
**Pull request by rubenwardy**
.nomedia files prevent images from appearing in the phone's gallery app and other places.

This file was present in the old way of extracting/building assets

## To do

- [x] Check this works recursively
- [x] Testing

## How to test

Use Minetest, check gallery app

Gallery apps may cache the pictures, and won't recheck `.nomedia`. It also expects `.nomedia` to be the first file, which may not happen [if the folder is already indexed](https://stackoverflow.com/questions/5895992/android-nomedia-is-ignored-images-still-appear-on-the-gallery) (crazy assumption?!) I needed to delete the Minetest folder and reextract the assets for this to work.

### CODE REVIEW

File: src/gui/touchscreengui.cpp [hunk 1]
```diff
@@ -1043,7 +1043,7 @@ void TouchScreenGUI::applyJoystickStatus()
 		if (i == 4 && !m_joystick_triggers_special1)
 			continue;
 
-		SEvent translated;
+		SEvent translated{};
```
Comment (MoNTE48) [inline src/gui/touchscreengui.cpp:1046]: 😂

**Comment by MoNTE48** [comment]
Does it work on Windows? If not, MultiCraft creates this file from Java.

**Comment by rubenwardy** [comment]
Not sure, it may be done in the Java. I can check. I'm not sure which way is better

Yes, this does work on Windows :)

File: src/gui/touchscreengui.cpp (hunk 1 above)
Comment (rubenwardy) [reply src/gui/touchscreengui.cpp:1046]: Not sure why this doesn't just have an empty diff?

**Comment by MoNTE48** [comment]
When worked - leave it as is.

**Pull request merged**